Lib9c/Action/DailyReward.cs

```diff
@@ -38,13 +38,13 @@ public override IAccountStateDelta Execute(IActionContext context)
                 throw new FailedLoadStateException($"{addressesHex}Aborted as the game config was failed to load.");
             }
 
-            if (avatarState.dailyRewardReceivedIndex < context.BlockIndex - gameConfigState.DailyRewardInterval)
+            if (context.BlockIndex - avatarState.dailyRewardReceivedIndex < gameConfigState.DailyRewardInterval)
             {
                 var sb = new StringBuilder()
                     .Append($"{addressesHex}Not enough block index since the last received daily reward.")
                     .Append(
-                        $" Expected: Equals or greater than {context.BlockIndex - gameConfigState.DailyRewardInterval}.")
-                    .Append($" Actual: {avatarState.dailyRewardReceivedIndex}");
+                        $" Expected: Equals or greater than ({gameConfigState.DailyRewardInterval}).")
+                    .Append($" Actual: ({context.BlockIndex - avatarState.dailyRewardReceivedIndex})");
```

Comment:
It's more natural for me to print `context.BlockIndex` because we're talking about block index (https://github.com/planetarium/lib9c/pull/579/files#diff-8c468fee48a7668caf870bd0919f71dce01be357858e6687ee5cb5784afd73dfR44).